homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -681,15 +681,15 @@ async def wake_word_detection(
             wake_word_output: dict[str, Any] = {}
         else:
             # Avoid duplicate detections by checking cooldown
-            last_wake_up = self.hass.data.get(DATA_LAST_WAKE_UP)
+            last_wake_up = self.hass.data[DATA_LAST_WAKE_UP].get(result.wake_word_id)
```

Comment:
This is incorrect. We should use entity ID + wake word ID as we cannot guarantee wake word IDs to be globally unique.